templates/etc/systemd/system/apt-daily-upgrade.timer.d/override.conf.j2

```diff
@@ -0,0 +1,13 @@
+[Timer]
+{% if apt_unattended_upgrades_upgrade_timer_replace is defined and apt_unattended_upgrades_upgrade_timer_replace|bool %}
```

Comment:
This seems a bit suspicious... it could end up like
```
OnCalendar=
OnCalendar=XXXX
```
Would this work too?
```
{% if apt_unattended_upgrades_upgrade_timer_replace is defined and apt_unattended_upgrades_upgrade_timer_replace|bool %}
OnCalendar={{ apt_unattended_upgrades_upgrade_timer_on_calendar | default("") }}
{% endif %}
```

Reply:
Yes, it is exactly the purpose.
The empty OnCalendar, erase/delete the current schedule.
The next one add the schedule.
Without the empty one, the 2 schedules will be merged
I think we have to keep the things like this